src/utils/index.ts

```diff
@@ -1,47 +0,0 @@
-import { getAddress } from '@ethersproject/address'
```

Comment:
This isn't a barrel file, look at `src/nft/queries/index.ts` for reference where the index file is a conglomerate of the various exports in the folder. This will allow all imports from this folder to just be from `utils` such as 
`import { isIOS } from 'utils'`

Reply:
I did, because there were issues caused by it unfortunately. I fixed 5-6 circular dependencies but there’s one more that’s actually non trivial to unroll, so I bailed out and reverted to non barrel.